public/app/plugins/datasource/cloudwatch/components/shared/Dimensions/Dimensions.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { fireEvent, render, screen, within } from '@testing-library/react';
+import { act, fireEvent, render, screen, within } from '@testing-library/react';
```

Comment:
Is there a warning if `act` isn't used?

If there are any warnings, you could probably fix them with using the async query methods https://testing-library.com/docs/dom-testing-library/api-async/#findby-queries